app/controllers/application_controller.rb

```diff
@@ -1,13 +1,13 @@
 require "application_responder"
 
 class ApplicationController < ActionController::Base
-  self.responder = ApplicationResponder
-  respond_to :html
-
   protect_from_forgery with: :exception
   before_action :set_locale
+  self.responder = ApplicationResponder
+  respond_to :html
```

Comment:
Why did you move the lines?

Reply:
I probably moved when the translation was not working. :) Moved back with fixup in sha `e728c8d`